src/core/Mixer.cpp

```diff
@@ -780,7 +787,7 @@ void Mixer::requestChangeInModel()
 
 	m_doChangesMutex.lock();
 	m_waitChangesMutex.lock();
-	if ( m_isProcessing && !m_waitingForWrite && !m_changesSignal )
+	if( m_isProcessing && !m_waitingForWrite && !m_changesSignal )
```

Comment:
Please revert or convert to the new style.
Revert:
```suggestion
	if ( m_isProcessing && !m_waitingForWrite && !m_changesSignal )
```
New style:
```suggestion
	if (m_isProcessing && !m_waitingForWrite && !m_changesSignal)
```